supportedBackends/local/src/test/scala/cromwell/backend/impl/local/BackendTestkitSpec.scala

```diff
@@ -0,0 +1,88 @@
+package cromwell.backend.impl.local
+
+import java.nio.file.FileSystems
+
+import akka.actor.ActorSystem
+import akka.testkit.TestActorRef
+import com.typesafe.config.ConfigFactory
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionFailedResponse, BackendJobExecutionFailedRetryableResponse, BackendJobExecutionResponse, BackendJobExecutionSucceededResponse}
+import cromwell.backend._
+import cromwell.backend.impl.local.TestWorkflows.TestWorkflow
+import cromwell.core.{WorkflowId, WorkflowOptions}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{Millis, Seconds, Span}
+import org.scalatest.{Matchers, Tag}
+import spray.json.{JsObject, JsValue}
+import wdl4s._
+import wdl4s.values.WdlValue
+
+object BackendTestkitSpec {
+  implicit val testActorSystem = ActorSystem("LocalBackendSystem")
+  object DockerTest extends Tag("DockerTest")
+
+}
+
+trait BackendTestkitSpec extends ScalaFutures with Matchers {
+  import BackendTestkitSpec._
+
+  val localFileSystem = List(FileSystems.getDefault)
+  val defaultBackendConfig = new BackendConfigurationDescriptor("local-config", ConfigFactory.load())
+  val defaultConfig = defaultBackendConfig.config.getConfig(defaultBackendConfig.configPath)
+
+  implicit val defaultPatience = PatienceConfig(timeout = Span(5, Seconds), interval = Span(500, Millis))
+
+  def testWorkflow(workflow: TestWorkflow) = {
+    val backend = localBackend(jobDescriptorFromSingleCallWorkflow(workflow.workflowDescriptor), workflow.config)
+    executeJobAndAssertOutputs(backend, workflow.expectedResponse)
+  }
+
+  def buildWorkflowDescriptor(wdl: WdlSource,
+                              inputs: Map[String, WdlValue] = Map.empty,
+                              options: WorkflowOptions = WorkflowOptions(JsObject(Map.empty[String, JsValue])),
```

Comment:
Hmm yeah, I like to keep it separate because this way you can easily reuse the same Wdl with different inputs without having to create a new `WorkflowSourceFiles`, but it's no big deal either way